app/src/main/java/org/mifos/selfserviceapp/api/services/ClientService.java

```diff
@@ -16,15 +17,23 @@
  */
 public interface ClientService {
 
+    String CLIENT_ID = "clientId";
```

Comment:
Checkstyle test was failing since string "clientId" was being repeated more than 4 times here, so I made it as a constant.
Interface constants are implicitly `public static final` so defining them as `static final` is redundant. Also, interfaces cannot have private modifiers.